### ISSUE TITLE
Cannot get diffs for files with source blocks

### ISSUE THREAD

**Issue by yantar92** [tecosaur/orgdiff#3]
I export setup for src blocks with `org-latex-listings` set to t.

Expected behaviour: can generate diffs with orgdiff

Observed: latexdiff does not play well with code listing environment and orgdiff and always fail to generate pdfs.

**Comment by tecosaur** [comment]
For what it's worth, this seems to work better with the engraved backend.

**Comment by yantar92** [comment]
tecosaur ***@***.***> writes:

> For what it's worth, this seems to work better with the engraved backend.

Not every publisher allows arbitrary LaTeX packages.

-- 
Ihor Radchenko // yantar92,
Org mode contributor,
Learn more about Org mode at <https://orgmode.org/>.
Support Org development at <https://liberapay.com/org-mode>,
or support my work at <https://liberapay.com/yantar92>

**Comment by tecosaur** [comment]
Indeed. I'm not sure what else can reasonably be done here though. Filter out source blocks?

**Comment by yantar92** [comment]
tecosaur ***@***.***> writes:

> Indeed. I'm not sure what else can reasonably be done here though. Filter out source blocks?

latexdiff appears to have toggles to configure this:
https://github.com/ftilmann/latexdiff/blob/1f9d07fd1b6edd914804442de0b396264311df8b/latexdiff#L115

-- 
Ihor Radchenko // yantar92,
Org mode contributor,
Learn more about Org mode at <https://orgmode.org/>.
Support Org development at <https://liberapay.com/org-mode>,
or support my work at <https://liberapay.com/yantar92>
